submissions/SergSenras/tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,54 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+  Complete the below for code reviewers' convenience:
+
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+*/
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Dweller {
+  constructor (species,name,gender,saying,friends) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  getInfo() {
+    var str = '';
+    for (var key in this) {
+      str += key + ': ' + this[key] + '; ';
+    }
+    return str;
+  }
+}
+
+class Animal extends Dweller {
+  constructor(species,name,gender,legs,saying,friends) {
+    super(species,name,gender,saying,friends);
+    this.legs = legs;
+  }
+}
+
+class Human extends Dweller {
+  constructor(species,name,gender,legs, hands,saying,friends) {
```

Comment:
Do we really need to ask a user for a species in this case?
`super` is a function that can accept literals as arguments just like any other function.
`super()` maps to `ParentClass.constructor()`. Number, order and whatever else between child class constructor and `super()` not necessarily coincide.

Reply:
```suggestion
class Human extends Dweller {
  constructor(name,gender,legs,hands,saying,friends) {
    super('human',name,gender,legs,saying,friends);
```
That's make sense! Thanks!